gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
@the-t-in-rtf, thanks for fixing all-tests.html. I've confirmed running all-tests.html in browser passes, but seeing numerous number of errors of `Not found http://localhost/testem.js` and `Not found http://localhost/coverage/client/coverageSender.js`. The screenshot with the errors at the start:

![screen shot 2017-05-24 at 9 56 11 am](https://cloud.githubusercontent.com/assets/572877/26408679/74a056a4-406c-11e7-9cfd-53c0d2615d6f.jpeg)

I'm not sure why individual tests pass on you. They simply would not pass in my browser while hitting errors of not found files. The execution would stop and the result is exactly as the screenshot I attached last time.